interview-questions.md

```diff
@@ -8,36 +8,47 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: This is not a big deal since Cohort will not have any foreign keys. When we generate Student model, each entry/student/row will have Cohort's id as their foreign key (cohort_id), since each student can only belong to one Cohort. Also we need to add belongs_to in our models folder ... Student < ApplicationRecord
 
-Researched answer:
+Researched answer: My researched answer provided nothing new for me to enter here. Unless im reading this incorrectly, only one model is generated at this point. The second model - Student - when generated, will have one additional column - cohort_id - as its foreign key.
```

Comment:
The questions was more asking in the event that your forgot to add the foreign key as you were creating the Student table.  In this case you would want to generate a migration file to update the schema.